rendering_resource_manager_service/deployment/rrm/populateRRM.txt

```diff
@@ -2,20 +2,7 @@
 # of the rendering resource manager.
 
 # Livre
-curl -curl --dump-header - -H "Accept:application/json" -H "Content-Type:application/json" -X POST --data '{"id": "livre", "command_line": "livre", "modules": "", "environment_variables": "", "process_rest_parameters_format": "--zeroeq-http-server ${rest_hostname}:${rest_port}", "scheduler_rest_parameters_format": "--zeroeq-http-server $SLURMD_NODENAME:${rest_port}", "graceful_exit": "True", "wait_until_running": "True"}' http://localhost:8383/rendering-resource-manager/v1/config/
-
-# RTNeuron
-curl -curl --dump-header - -H "Accept:application/json" -H "Content-Type:application/json" -X POST --data '{ "id": "rtneuron", "command_line": "rtneuron-app.py", "modules": "", "environment_variables": "", "process_rest_parameters_format": "--zeroeq-http-server ${rest_hostname}:${rest_port}", "scheduler_rest_parameters_format": "--zeroeq-http-server $SLURMD_NODENAME:${rest_port} ", "graceful_exit": "True", "wait_until_running": "True" }' http://localhost:8383/rendering-resource-manager/v1/config/
+curl -curl --dump-header - -H "Accept:application/json" -H "Content-Type:application/json" -X POST --data '{"id": "livre", "command_line": "livre", "modules": "BBP/viz/latest", "environment_variables": "", "process_rest_parameters_format": "--zeroeq-http-server :${rest_port}", "scheduler_rest_parameters_format": "--zeroeq-http-server :${rest_port}", "project": "proj3", "queue": "interactive", "exclusive": true, "nb_nodes": 1, "nb_cpus": 16, "nb_gpus": 1, "graceful_exit": true, "wait_until_running": true, "name": "Livre", "description": "Large scale Interactive Volumen REnderer"}' http://localhost:${rrm_service_port}/${rrm_service_name}/config/
```

Comment:
Yes, this is definitely a typo, or a volume in an undefined natural language :-). Thanks. Fixed.